### PR TITLE
DPP-1336 Fix ContractAgreement WebJob in Prod

### PR DESCRIPTION
**Pull request by lazcool**
This should fix the webjob failing in prod with the message 'Could not load file or assembly 'System.Net.Http, Version=4.1.1.2'.

Can't be sure because the web job ran on my dev machine anyway.

This switches the reference in the webjob to use the package version rather than the framework version.

I've checked the module loaded is v4.6.25705.01, which maps to the 4.1.1.2 (& 4.1.1.0) reported version (package v4.3.3).

### CODE REVIEW

File: src/SFA.DAS.PAS.ContractAgreements.WebJob/Program.cs [hunk 1]
```diff
@@ -28,6 +28,7 @@ static void Main()
             {
                 ILog exLogger = new NLogLogger();
                 exLogger.Error(ex, "Error running ContractAgreements WebJob");
+                throw;
```
Comment (tom-gough) [inline src/SFA.DAS.PAS.ContractAgreements.WebJob/Program.cs]: this isn't a package update, is it intended?

File: src/SFA.DAS.PAS.Host.WebJob/Web.config [hunk 2]
```diff
@@ -8,6 +8,6 @@
     <!--the targetFramework mismatch is valid (https://stackoverflow.com/questions/40170567/what-do-the-targetframework-settings-mean-in-web-config-in-asp-net-mvc)-->
     <!--but should we update the httpRuntime targetFramework?-->
     <compilation debug="true" targetFramework="4.6.2"/>
-    <httpRuntime targetFramework="4.5"/>
+    <httpRuntime targetFramework="4.6.2"/>
```
Comment (tom-gough) [inline src/SFA.DAS.PAS.Host.WebJob/Web.config]: is this an intended change?

Comment (lazcool) [reply src/SFA.DAS.PAS.Host.WebJob/Web.config]: Yea, we've upgraded the project to 4.6.2, so this is saying we want it to run under 4.6.2

**Comment by satya-sanikommu** [comment]
Review and testing completed. WebJob is running successfully on AT, TEST & TEST2. Ready to merge

**Pull request merged**